### ISSUE TITLE
Extended error codes (MS-WEBDAVE)

### ISSUE THREAD

**Issue by fubar-coder** [FubarDevelopment/WebDavServer#15]
The documentation can be downloaded from Microsoft: [MS-WEBDAVE](https://msdn.microsoft.com/en-us/library/office/cc313126.aspx).

The extended error codes are returned using the `X-MSDAVEXT_ERROR` header. A description of the header format can be found in the [MS-WDV](https://msdn.microsoft.com/en-us/library/cc250046.aspx) documentation, chapter 2.2.3.

**Comment by pizixie** [comment]
Is it supported???

**Comment by fubar-coder** [comment]
No, not yet. It only implements the standard as defined by the RFCs for WebDAV class 1 and 2.
